lib/views/help/privacy.html.erb

```diff
@@ -20,20 +20,30 @@
 
 <dt id="email_address">Who gets to see my email address? <a href="#email_address">#</a> </dt>
 
-<dd><p>We will not disclose your email address to anyone unless we are obliged to by law,
-or you ask us to. This includes the public authority that you are sending a
-request to.  They only get to see an email address
-@<%= AlaveteliConfiguration::incoming_email_domain %> which is specific to that request.  </p>
+<dd>
+
+<p>We will not disclose your email address to anyone unless we are obliged to by law,
+or you ask us to. </p>
+
+<p>This includes the public authority that you are sending a
+request to.  When you make a request to an authority, they will receive an email from a unique
+@<%= AlaveteliConfiguration::incoming_email_domain %> address. </p>
+
```

Comment:
Why did you split up the above into separate paragraphs? Since they're related sentences they should be part of the same paragraph shouldn't they?